.github/workflows/spellcheck.yaml

```diff
@@ -14,4 +14,4 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v4
       - name: Spell check
-        uses: crate-ci/typos@v1.24.6
+        uses: crate-ci/typos@v1
```

Comment:
we don't want to target a wide range, as it may spontaneously start failing CI runs due to a previously unchecked mistake

Reply:
Okay i change for the specify latest based in how Action dont throw any with that version `[1.32.0](https://github.com/crate-ci/typos/releases/tag/v1.32.0)`